rewrite-gradle-tooling-model/plugin/src/main/java/org/openrewrite/gradle/toolingapi/ToolingApiOpenRewriteModelPlugin.java

```diff
@@ -71,4 +73,12 @@ public Object buildAll(String modelName, Project project) {
             }
         }
     }
+
+    interface MavenRepositoryMixin {
+        @JsonProperty(access = JsonProperty.Access.READ_WRITE)
+        String getUsername();
+
+        @JsonProperty(access = JsonProperty.Access.READ_WRITE)
+        String getPassword();
+    }
```

Comment:
In the `MavenRepository`, we prevent username and password from being written into the LST while here we are attempting to transfer it from the Gradle process to the tooling API where it is acceptable to serialize it.